modules/pipeline/spec/pipeline_extra.go

```diff
@@ -198,3 +198,13 @@ func (extra *PipelineExtra) GetOrgName() string {
 	}
 	return ""
 }
+
+func (extra *PipelineExtra) GetOrgID() string {
+	if extra.NormalLabels != nil {
```

Comment:
get from filterLabels and normalLabels by order.
and adjust method to get org name.